.babelrc

```diff
@@ -6,16 +6,52 @@
     "transform-class-properties",
     "array-includes",
     [
-        "transform-runtime", {
-            "polyfill": false,
-            "regenerator": true
-        }
+      "transform-runtime", {
+        "polyfill": false,
+        "regenerator": true
+      }
     ]
   ],
   "presets": [
-	  "flow",
-	  "es2016",
-	  "es2017",
-	  "react"
-  ]
+    "flow",
+    "react",
+    ["env", {
+      "targets": {
+        "node": "current"
+      }
+    }]
+  ],
+  "env": {
+    "browser": {
```

Comment:
I've created a babel environment specifically for the browser in place of having a separate config for the client directory. As long as `BABEL_ENV` is set to `browser` when building the site code (which I've done in the client JS build scripts), we'll build for our target browsers.